Core/Systems/Affixes/ItemTypes/ArmorAffixes/DefenseAffixes.cs

```diff
@@ -1,52 +1,41 @@
 ﻿namespace PathOfTerraria.Core.Systems.Affixes.ItemTypes.ArmorAffixes;
 
-internal class DefenseAffixes
+internal class DefenseItemAffix : ItemAffix
 {
-	internal class DefenseItemAffix : ItemAffix
+	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
 	{
-		public override ItemType PossibleTypes => ItemType.Armor;
-		public override void ApplyAffix(EntityModifier modifier, PoTItem item)
-		{
-			modifier.Defense.Base += 1 + Value * 5f + item.ItemLevel / 50f;
-		}
+		modifier.Defense.Base += Value;
 	}
+}
 
-	internal class EnduranceItemAffix : ItemAffix
-	{ // this seems to need some modification
-		public override ItemType PossibleTypes => ItemType.Armor;
-		public override void ApplyAffix(EntityModifier modifier, PoTItem item)
-		{
-			modifier.DamageReduction.Base += (float)Math.Truncate((Value * 5 + item.ItemLevel / 50) * 10) / 10;
-		}
+internal class EnduranceItemAffix : ItemAffix
+{
+	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
+	{
+		modifier.DamageReduction.Base += Value;
 	}
+}
 
-	internal class ResistanceHelmetAffix : ItemAffix
+internal class ResistanceHelmetAffix : ItemAffix
+{
+	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
 	{
-		public override ItemType PossibleTypes => ItemType.Helmet;
-
-		public override void ApplyAffix(EntityModifier modifier, PoTItem item)
-		{
-			modifier.DebuffResistance *= 1 - (float)Math.Truncate((Value * 2 + item.ItemLevel / 450) * 5) / 300f;
-		}
+		modifier.DebuffResistance *= Value;
 	}
+}
 
-	internal class BuffBoostHelmetAffix : ItemAffix
+internal class BuffBoostHelmetAffix : ItemAffix
+{
+	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
 	{
-		public override ItemType PossibleTypes => ItemType.Helmet;
-
-		public override void ApplyAffix(EntityModifier modifier, PoTItem item)
-		{
-			modifier.BuffBonus *= 1 + (float)Math.Truncate((Value * 2 + item.ItemLevel / 450) * 5) / 300f;
-		}
+		modifier.BuffBonus *= Value;
 	}
+}
 
-	internal class ThornyArmorAffix : ItemAffix
+internal class ThornyArmorAffix : ItemAffix
+{
+	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
 	{
-		public override ItemType PossibleTypes => ItemType.Armor;
-
-		public override void ApplyAffix(EntityModifier modifier, PoTItem item)
-		{
-			modifier.ReflectedDamageModifier += Value * 0.1f;
-		}
+		modifier.ReflectedDamageModifier += Value * 0.1f;
```

Comment:
Min and max are floats so why the *0.1 here?

Reply:
I just pushed a very large commit that fixed exactly that xD